lib/matplotlib/tests/test_animation.py

```diff
@@ -164,7 +164,38 @@ def animate(i):
         except UnicodeDecodeError:
             pytest.xfail("There can be errors in the numpy import stack, "
                          "see issues #1891 and #2679")
+        try:
+            length = len(fig.animations)
+            assert(length != 0)
+        except AttributeError:
```

Comment:
I don't think the explicit `try..except` adds much here, just the bare assert is good enough in both places.